Updates container_cluster to set enable_legacy_abac to false by default

mrparkers opened the pull request:
Discussion in #1116.

GKE disables legacy ABAC by default in 1.8, and the current default GKE version (as of opening this PR) is 1.8.8.  This means that creating a GKE cluster using the cloud console and leaving all values as default will result legacy ABAC being disabled, but creating a GKE cluster using Terraform and leaving all values as default will keep it enabled.  This PR disables legacy ABAC by default so the default behavior is more consistent with the cloud console.

Test:

```
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccContainerCluster_withDefaultLegacyAbac -timeout 120m
?   	github.com/terraform-providers/terraform-provider-google	[no test files]
=== RUN   TestAccContainerCluster_withDefaultLegacyAbac
--- PASS: TestAccContainerCluster_withDefaultLegacyAbac (243.16s)
```

danawillow commented on google/resource_container_cluster_test.go:
```diff
@@ -368,6 +368,29 @@ func TestAccContainerCluster_withLegacyAbac(t *testing.T) {
+				Config: testAccContainerCluster_defaultLegacyAbac(acctest.RandString(10)),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_container_cluster.default_legacy_abac", "enable_legacy_abac", "false"),
+				),
```
Mind adding an import step here as well? (Take a look at the other tests in this file for examples)

mrparkers commented:
@danawillow I addressed your comment.

```
TF_ACC=1 go test $(go list ./... |grep -v 'vendor') -v -run=TestAccContainerCluster_withDefaultLegacyAbac -timeout 120m
?   	github.com/terraform-providers/terraform-provider-google	[no test files]
=== RUN   TestAccContainerCluster_withDefaultLegacyAbac
--- PASS: TestAccContainerCluster_withDefaultLegacyAbac (233.43s)
PASS
ok  	github.com/terraform-providers/terraform-provider-google/google	233.458s
```

danawillow commented:
Thanks a bunch!
